pkgs/dart_mcp_server/test/tools/flutter_launcher_test.dart

```diff
@@ -460,7 +460,6 @@ void main() {
             'TimeoutException',
           ]),
         );
-        test.expect(mockProcessManager.killedPids, [processPid]);
```

Comment:
You should be able to look for the mocked processes here and then check if they were killed